.github/workflows/presubmit.yml

```diff
@@ -19,6 +22,7 @@ jobs:
             -Dorg.gradle.internal.http.socketTimeout=60000                  \
             -Dorg.gradle.internal.repository.max.retries=20                 \
             -Dorg.gradle.internal.repository.initial.backoff=500            \
+            --no-configuration-cache                                        \
```

Comment:
Are we disabling configuration to speed up CI?

Reply:
There are 2 reasons this needs to be disabled:
1. Currently, the `buildOnServer` task fails when restored from the configuration cache. This happens when I run the build locally: https://ge.androidx.dev/s/zist7pgwj3qfy/console-log#L303
2. The configuration-cache state contains links directly into the `caches/jars-9` and `caches/transforms-3` directories that are currently being excluded from the Gradle User Home save/restore.

We can fix the first issue, but until we are sure we can remove the cache excludes we cannot use configuration caching. I'll add a comment to this effect.

Note that there isn't yet a way to tell `gradle-build-action` to avoid save/restore of the configuration-cache state: if the configuration-cache is written in a build invocation, the action will save it and restore on the next run. So `--no-configuration-cache` is the best way to avoid this problem.

Reply:
Comment added to workflow file.